src/main/java/com/iotiq/application/service/CustomerService.java

```diff
@@ -115,12 +120,18 @@ public void createCustomer(User currentUser) {
         customer.setMedicalData(new MedicalData());
         customer.setSizeInfo(new SizeInfo());
         customerRepository.save(customer);
```

Comment:
we dont need this anymore. We should have removed this when we added the creation in the filter